src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -189,19 +233,83 @@ export class QuerySummary extends Component {
     }
   }
 
+  private updateQueryTagsInNoResultsContainer() {
```

Comment:
The new logic is from line 236 to 261.
Line 236 : updateQueryTagsInNoResultsContainer() : makes possible the usage of the queryTag (${query}) in the code editor of the QuerySummary component.
Line 250 : parseQueryTags() : the new logic for parsing the queryTag from a string.

Reply:
About the vulnerable to any XSS attack : 
On line 246 : this is pretty much the same behaviour as before. So I should check with a UT.
On line 256 : does the function .replace is considered safe?